mail/tasks_test.py

```diff
@@ -10,20 +10,32 @@
 from mail import tasks
 from mail.models import NotificationEmail
 from ui.constants import VideoStatus
-from ui.factories import VideoFactory
+from ui.factories import VideoFactory, CollectionFactory, MoiraListFactory
 
 pytestmark = pytest.mark.django_db
 
 
 # pylint: disable=protected-access
 
 
-def test_get_recipients_for_video():
+@pytest.fixture(autouse=True)
+def mocker_defaults(mocker):
+    """
+    Sets default settings to safe defaults
+    """
+    mocker.patch('mail.tasks.has_common_lists', return_value=False)
+
+
+def test_get_recipients_for_video(mocker):
     """
     Tests the _get_recipients_for_video api
     """
-    video = VideoFactory()
-    assert tasks._get_recipients_for_video(video) == [video.collection.owner.email]
+    lists = [MoiraListFactory(), MoiraListFactory()]
+    video = VideoFactory(collection=CollectionFactory(admin_lists=lists))
+    list_emails = ['{}@mit.edu'.format(mlist.name) for mlist in video.collection.admin_lists.all()]
+    assert tasks._get_recipients_for_video(video) == list_emails + [video.collection.owner.email]
```

Comment:
This assertion seems to depend on `has_common_lists` returning `False`. Even though you've set the default return value to `False`, I think we should explicitly set the return value before this assertion. It's not clear from within the test case that the assertion depends on that `False` value

Reply:
OK